Assignments/10 - Culminating/README.md

```diff
@@ -0,0 +1,96 @@
+**The culminating is worth 30% of your final mark.**
+
+Task(s)
+-------
+
+
+###Part 1
+
+The following survey will be conducted during class time to you and your ICS 3C and 3U peers. 
+
+  1. On a scale from 0 to 10, how fun is programming?
+  2. What percentage of the software development process would you estimate is spent writing code?
+  3. Name one aspect of being a good digital citizen.
+  4. Name one job that you can pursue with a background in computer science.
+  5. Briefly explain why programming is a useful skill.
+
+You will be given the results in the form of a spreadsheet. Using the knowledge you've acquired from this course, store the results in **two different ways**.
```

Comment:
Do you have a survey ready for this, or do you want me to make a Google Form or something?